### PR TITLE
Validate contains

### PR DESCRIPTION
**Pull request by markiantorno**
Grahame, do not merge this yet please.

### CODE REVIEW

File: org.hl7.fhir.validation/src/main/java/org/hl7/fhir/validation/BaseValidator.java [hunk 1]
```diff
@@ -1,84 +1,9 @@
...
-
-import java.text.DateFormat;
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.HashMap;
-
-/*
-  Copyright (c) 2011+, HL7, Inc.
```
Comment (dotasek) [inline org.hl7.fhir.validation/src/main/java/org/hl7/fhir/validation/BaseValidator.java:13]: The copyright notice shouldn't be deleted.

Comment (markiantorno) [reply org.hl7.fhir.validation/src/main/java/org/hl7/fhir/validation/BaseValidator.java:13]: Dunno how that happened.
Fixed.

**Review by dotasek — APPROVED** [review]

**Comment by markiantorno** [comment]
The pipeline passed....? But it has marked this as fail?
Results here: https://dev.azure.com/fhir-pipelines/fhir-core-library/_build/results?buildId=5011&view=results

**Comment by dotasek** [comment]
I saw that; Azure and the dedicated GitHub checks page all showed green, but the commends landing held the failed result. Annoying, but not terribly worrying.

**Pull request merged**